frontend/src/Components/AlgInputs/AlgInputsLoadForecast.js

```diff
@@ -96,51 +96,54 @@ class AlgInputsLoadForecast extends Component {
     };
 
     update = (field, event) => {
-        this.setState({ [field]: event.currentTarget.value })
+        this.setState({ [field]: event.currentTarget.value });
     };
 
-    saveResults = () => {
+    saveResults = async() => {
+        // check if current load forecast profile already exists before saving
+        const config_res = await axios.get(`http://127.0.0.1:8000/api/config/load_forecast?config_name=${this.state.config_name}`);
 
-        // change var name
-        const postData = {
-            config_name: this.state.config_name,
-            aggregation_level: this.state.aggregation_level,
-            num_evs: parseInt(this.state.num_evs),
-            county: this.state.county_choice,
-            fast_percent: parseFloat(this.state.fast_percent),
-            work_percent: parseFloat(this.state.work_percent),
-            res_percent: parseFloat(this.state.res_percent),
-            l1_percent: parseFloat(this.state.l1_percent),
-            public_l2_percent: parseFloat(this.state.public_l2_percent),
-            res_daily_use: parseFloat(this.state.res_daily_use),
-            work_daily_use: parseFloat(this.state.work_daily_use),
-            fast_daily_use: parseFloat(this.state.fast_daily_use),
-            rent_percent: parseFloat(this.state.rent_percent),
-            res_l2_smooth: this.state.res_l2_smooth,
-            week_day: this.state.week_day,
-            publicl2_daily_use: parseFloat(this.state.publicl2_daily_use),
-            mixed_batteries: this.state.mixed_batteries,
-            timer_control: this.state.timer_control,
-            work_control: this.state.work_control,
-        }
-
-        const postUrl = `${ serverUrl }/load_forecast_runner`;
+        // if the CBA input relationship doesn't exist, insert new CBA input table rows to db
+        if(config_res.data.length === 0){
```

Comment:
0 coerces to false, you can just simplify this to `if(!config_res.data.length)`

Reply:
Yes will change thanks!